src/fsharp/service/service.fsi

```diff
@@ -352,6 +356,10 @@ type public FSharpProjectOptions =
       /// If two sets of options both have stamps, then they are considered equal
       /// if and only if the stamps are equal
       Stamp: int64 option
+
+      /// When true, error messages will include suggestions for parsing errors based on what the compiler knows.
+      /// This is an expensive operation that should be false outside of code fixes, command-line builds, and in-IDE builds.
+      SuggestNamesForErrors: bool
```

Comment:
I doubt it should be in project options as there's probably no scenario where we want to suggest it for one project or script and not suggest it for others. As this is mostly an environment difference thing (IDE analysis vs build time) I think it should be a `FSharpChecker` parameter instead.

Reply:
@auduchinok Yes, that sounds more appropriate. It's probably also a bit less churn. Although having some ability to have it off in one project and on in another (or in script files) may be interesting, having it be a global toggle is consistent with the rest of the IDE features.